
<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Sairam Behera</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
    <link rel="stylesheet" href="site.css">
  </head>
  <body>
    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-light">
      <a class="navbar-brand" href="#">Sairam Behera</a>
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>
      <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
        <div class="navbar-nav">
          <a class="nav-item nav-link" href="#bio">Bio</a>
          <a class="nav-item nav-link" href="#publications">Publications</a>
          <a class="nav-item nav-link" href="#softwares">Softwares</a>
          <a class="nav-item nav-link" href="#posters">Posters</a>
          <a class="nav-item nav-link" href="#talks">Talks</a>
          <a class="nav-item nav-link" href="#teachings">Teachings</a>
          <a class="nav-item nav-link" href="#contact">Contact</a>
        </div>
      </div>
    </nav>
    <div class="container">
      <div class="row">
        <div class="col-lg-3">
          <img class="mx-auto d-block my-1" src="sairam.png" alt="Sairam Behera">
          <h1 class="text-center mb-2">Sairam Behera</h1>
          <div class="text-muted professorship mb-4">
            Post-doctoral Research Fellow<br>
            <!--Microsoft and Sloan Fellow<br> -->
		  <a href="https://www.hgsc.bcm.edu/">Human Genome Sequencing Center </a><br>
		  <a href="https://www.bcm.edu/">Baylor College of Medicine</a></div>
        </div>
        <div class="col-lg-1"></div>
        <div class="col-lg-8">
          <h2 id="bio">Bio</h2>
          <p>Sairam Behera's research interests are in the areas of graph theory, algorithms, bioinformatics and next-generation sequencing data analysis. In particular, he has been working on the development and analysis of algorithms for computational biology. He received his B.Tech in Computer Science and Engineering from <a href="http://www.nitrkl.ac.in/">NIT Rourkela (India)</a>, M.S. from <a href="http://www.utdallas.edu/"> University of Texas at Dallas</a>  and Ph.D. from <a href="https://www.unl.edu/"> University of Nebraska-Lincoln</a>. 
          <!-- <p><a href="CV.pdf">Full CV</a></p> -->

          <h2 id="publications">Publications</h2>
          <a href="http://dblp.uni-trier.de/pers/hd/b/Behera:Sairam"> DBLP</a> <br>
     <a href="https://scholar.google.com/citations?user=NVwd1o8AAAAJ&hl=en"> Google Scholar</a><br><br>
    
		<b>Journals:</b><br>
   <ul>
	   <li> <b>S. Behera</b>, J. Belyeu, X. Chen, L. F Paulin, N. Q. Nguyen, E. Newman, M. Mahmoud, V. K. Menon, Q. Qi, P. Joshi, S. Marcovina, M. Rossi, E. Roller, J. Han, V. Onuchic, C. L Avery, C. M Ballantyne, C. J Rodriguez, R. C. Kaplan, D. M. Muzny, G. A. Metcalf, R. Gibbs, B. Yu, E. Boerwinkle, M. A. Eberle, F. J. Sedlazeck; <i>Identification of allele-specific KIV-2 repeats and impact on Lp (a) measurements for cardiovascular disease risk; </i> bioRxiv; doi: <a href=https://doi.org/10.1101/2023.04.24.538128>10.1101/2023.04.24.538128</a></i>
	   <li> C. Chin, <b>S. Behera</b>, A. Khalak, F. J. Sedlazeck, J. Wagner, J. M. Zook; <i> Multiscale Analysis of Pangenome Enables Improved Representation of Genomic Diversity For Repetitive And Clinical Relevant Genes</i>; bioRxiv; doi: <a href=https://doi.org/10.1101/2022.08.05.502980>10.1101/2022.08.05.502980</a></li> [Accepted in <i> Nature Methods </i>]
	   <li> L. Gan, K. Park, J. Chai, E. M. Updike, H. Kim, A. Voshall, <b>S. Behera</b>, X. Yu, Y. Cai, C. Zhang, M. A. Wilson, J. P. Mower, E. N. Moriyama, C. Zhang, S. Kaewsuwan, Q. Liu, J. Shanklin, E. B. Cahoon; <i>Divergent evolution of extreme production of variant plant monounsaturated fatty acids</i>; PNAS, Volume 119, Issue 30 (2022); doi: <a href=https://doi.org/10.1073/pnas.2201160119>10.1073/pnas.2201160119</a></li>
	   <li> C. Chin, <b>S. Behera</b>, G. Metcalf, R. A. Gibbs, E. Boerwinkle, F. J. Sedlazeck; <i>A pan-genome approach to decipher variants in the highly complex tandem repeat of LPA</i>; bioRxiv; doi: <a href=https://doi.org/10.1101/2022.06.08.495395>10.1101/2022.06.08.495395</a> </li>
	   <li> M. Smolka, L. F. Paulin, C. M. Grochowski, M. Mahmoud, <b>S. Behera</b>, M. Gandhi, K. Hong, D. Pehlivan, S. W. Scholz, C. MB Carvalho, C. Proukakis, F. J. Sedlazeck; <i>Comprehensive structural variant detection: from mosaic to population-level</i>; bioRxiv; doi: <a href=https://doi.org/10.1101/2022.04.04.487055>10.1101/2022.04.04.487055</a> </li>
           <li> <b>S. Behera</b>, J. LeFaive, P. Orchard, M. Mahmoud, L. F. Paulin, J. Farek, D. C. Soto, S. C. J. Parker, A. V. Smith, M. Y. Dennis, J. M. Zook, F. J. Sedlazeck; <i>FixItFelix: improving genomic analysis by fixing reference errors</i>; Genome Biology, Volume 24, Issue 31 (2023); doi: <a href=https://doi.org/10.1186/s13059-023-02863-7>10.1186/s13059-023-02863-7</a> </li>
	   <li> A. Voshall, <b>S. Behera</b>, X. Li, X. Yu, K. Kapil, J. S. Deogun, J. Shanklin, E. B Cahoon, E. N. Moriyama; <i>A consensus-based ensemble approach to improve transcriptome assembly</i>; BMC Bioinformatics 22, 513 (2021); doi: <a href=https://doi.org/10.1186/s12859-021-04434-8>10.1186/s12859-021-04434-8</a></li>
           <li> X. Lai*, <b>S. Behera* </b>, Z. Liang, Y. Lu, J. S. Deogun, J. C. Schnable; <i>STAG-CNS: An Order-Aware Conserved Noncoding Sequences Discovery Tool for Arbitrary Numbers of Species</i>; Molecular Plant, Volume 10, Issue 7 (2017); doi: <a href=https://doi.org/10.1016/j.molp.2017.05.010>10.1016/j.molp.2017.05.010</a> [*joint first author]</li>
   </ul>
<br>
		
		<b>Conferences:</b> <br>
     <ul>
	     
	     <li> <b>S. Behera</b>, J. S. Deogun, E. N. Moriyama; <i>MinIsoClust: Isoform clustering using minhash and locality sensitive hashing</i>; ACM-BCB 2020, pp. 1-7 </li>
         <li> <b>S. Behera</b>, S. Gayen, J. S. Deogun and N. V. Vinodchandran; <i>KmerEstimate: A Streaming Algorithm for Estimating k-mer Counts with Optimal Space Usage</i>; ACM-BCB 2018, pp. 438-447 </li>
         <li> <b>S. Behera</b>, J. S. Deogun, X. Lai and J. C. Schnable, <i>DiCE: Discovery of Conserved Noncoding Sequences Efficiently</i>; IEEE BIBM 2017, pp. 79-82 </li>
	 <li> <b>S. Behera </b>, A. Voshall, J. S. Deogun and E. N. Moriyama, <i>Performance comparison and an ensemble approach of transcriptome assembly</i>;  IEEE BIBM 2017, pp. 2207-2210 </li>
	 <li>N. Pavlovikj, K. Begcy, <b>S. Behera</b>, M. Campbell, H. Walia, J. S. Deogun; <i>Analysis of transcriptome assembly pipelines for wheat</i>; IEEE BIBM 2016, pp. 137-140 </li>
	 <li>N. Pavlovikj, K. Begcy, <b>S. Behera</b>, M. Campbell, H. Walia, J. S. Deogun; <i>Comparing and optimizing transcriptome assembly pipeline for diploid wheat</i>; ACM-BCB 2014, pp. 603-604 </li>
	 <li>N. Pavlovikj, K. Begcy, <b>S. Behera</b>, M. Campbell, H. Walia, J. S. Deogun; <i>A Comparison of a Campus Cluster and Open Science Grid Platforms for Protein-Guided Assembly Using Pegasus Workflow Management System</i>; IPDPS 2014, pp. 546-555 </li>
	 <li>N. Pavlovikj, K. Begcy, <b>S. Behera </b>, M. Campbell, H. Walia, J. S. Deogun; <i>Evaluating Distributed Platforms for Protein-Guided Scientific Workflow</i> XSEDE 2014, 38:1-38:8 </li>
	 <li> <b>S. Behera </b>, O. Daescu, L. Papiez; <i>Optimal delivery of volumetric modulated arc therapy (VMAT) for moving target</i>; IEEE BIBM 2011, pp. 297-304 </li>
    </ul>
   <br>
   
		<b>Book chapters</b>:
   <ul>
	   <li> <b>S. Behera</b>, A. Voshall, E. N. Moriyama; <i>Plant transcriptome assembly: review and benchmarking</i>; Bioinformatics, Exon Publications; Chapter 7, pp. 109-130 (2021); doi: <a href=https://doi.org/10.36255/exonpublications.bioinformatics.2021.ch7>10.36255/exonpublications.bioinformatics.2021.ch7</a></li>
	<li> <b>S. Behera</b>, J. S. Deogun, E. N. Moriyama; <i>MinCNE: Identifying Conserved Noncoding Elements Using Min-Wise Hashing</i>; Advances in Computer Vision and Computational Biology, Transactions on Computational Science and Computational Intelligence, Springer (2021); doi: <a href=https://doi.org/10.1007/978-3-030-71051-4_53>10.1007/978-3-030-71051-4_53</a> </li> 
		</ul>
		
          <h2 id="softwares">Softwares</h2>
          <ul>
            <li><strong>KmerEstimate</strong>: A Streaming Algorithm for Estimating k-mer Counts with Optimal Space Usage. <a href="https://github.com/srbehera11/KmerEstimate">https://github.com/srbehera11/KmerEstimate</a></li>
            <li><strong>STAG_CNS</strong>: An order-aware conserved noncoding sequence discovery tool for arbitrary numbers of species. <a href="https://github.com/srbehera11/stag-cns">https://github.com/srbehera11/stag-cns/</a></li>
            <li><strong>DiCE</strong>: Finding CNS in similar species. <a href="https://github.com/srbehera11/DiCE">https://github.com/srbehera11/DiCE</a></li>
          </ul>
          <h2 id="posters">Posters</h2>
          <ul>
		  <li> <i>Fixing falsely duplicated and collapsed regions of the GRCh38 reference genome</i>; [ASHG 2022, Reviewers' choice abstract] </li>
		  <li> <i>Identification of allele-specific KIV-2 repeats among multi-ethnic groups and association with LP(a) measurements</i>; ASHG 2021 </li>
		  <li> <i>Comprehensive genomics at scale using DRAGEN pipeline</i>; CSHL Genome Informatics, 2021 </li>
		  <li><i>Towards Improving Transcriptome Assemblies</i> UNL Plant Science Retreat 2018</li>
		  <li><i>Performance comparison and an ensemble approach of transcriptome assembly</i>; IEEE BIBM 2017</li>
		  <li><i>STAG-CNS: Conserved non-coding sequence discovery tool</i>; UNL Plant Science Retreat 2016</li>
		  <li> <i>Transcriptome analysis of drought responses in wheat</i>; UNL Plant Science Retreat 2014</li>
  </ul>
           <h2 id="talks">Talks</h2>
           <ul>
		   <li> Bio-IT World Conference 2023 </li>
	 	   <li> Labroots Genetic Week 2023 </li>
		   <li> HGSC seminar talk: <i>Towards fixing GRCh38 reference genome</i>; Human Genome Sequencing Center, Baylor College of Medicine </li>
		   <li> PSI seminar talk: <i>Towards Improving Transcriptome Assemblies</i>; Center for plant science innovation, University of Nebraska-Lincoln</li>
           </ul>
        
          <h2 id="teachings">Teachings</h2>
		I was an instructor or Teaching Assistant for the following courses at University of Nebraska-Lincoln
          <ul>
      <li><a href="https://bulletin.unl.edu/courses/CSCE/924">CSCE 924</a>: Graph Algorithms </li>
       <li><a href="https://bulletin.unl.edu/courses/CSCE/923">CSCE 923</a>: Development and Analysis of Efficient Algorithms</li>

        <li><a href="http://bulletin.unl.edu/courses/CSCE/423">CSCE 823</a>: Design and Analysis of Algorithms</li>
       <li><a href="http://bulletin.unl.edu/courses/CSCE/251K">CSCE 251K</a>: C Programming</li>
        <li><a href="http://bulletin.unl.edu/courses/CSCE/155H">CSCE 155H</a>: Computer Science I Honors</li>
        <li><a href="http://bulletin.unl.edu/courses/CSCE/155E">CSCE 155E</a>: Computer Science I</li>
    
</ul>
          <h2 id="contact">Contact</h2>
          <p>Email: <kbd>Sairam[DOT]Behera[AT]bcm[DOT]edu</kbd><br>
          <address>
            Baylor College of Medicine <br>
            1 Baylor Plaza, Houston, TX 77030<br>
          </address>

          <a class="twitter-timeline" data-height="600" href="https://twitter.com/srbehera11">Tweets by Sairam</a> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>

        </div>
      </div>
    </div>
  </body>
</html>

